system/lib/wasmfs/file_table.cpp

```diff
@@ -117,6 +119,16 @@ std::shared_ptr<Directory> getRootDirectory() {
   return rootDirectory;
 }
 
+std::shared_ptr<File> getCWD() {
+  if (cwd) {
+    return cwd;
+  } else {
+    return getRootDirectory();
+  }
+};
+
+void setCWD(std::shared_ptr<File> directory) { cwd = directory; };
```

Comment:
If these are the only two methods accessing `cwd` then it could be defined right next to them, which would make reading the code easier.